src/main/java/com/minecraftabnormals/upgrade_aquatic/core/events/EntityEvents.java

```diff
@@ -181,6 +184,16 @@ public static void onPlayerMount(EntityMountEvent event) {
 		}
 	}
 	
+    @SubscribeEvent
+    public static void onDrownedPoseChange(EntityEvent.EyeHeight event) {
+    	Entity entity = event.getEntity();
+        if (entity instanceof DrownedEntity && event.getPose() == Pose.SWIMMING) {
+            DrownedEntity drowned = (DrownedEntity) entity;
+            event.setNewHeight(0.2F);
+            drowned.size = drowned.isChild() ? EntitySize.flexible(0.7F, 0.6F) : EntitySize.flexible(0.6F, 0.6F);
```

Comment:
This is redundant. Why not just do `EntitySize.flexible(drowned.isChild() ? 0.6F : 0.7F, 0.6F)` instead of creating two instances separately?